keyboards/pabile/p20/keymaps/default/keymap.c

```diff
@@ -0,0 +1,41 @@
+#include QMK_KEYBOARD_H
+#ifdef BACKLIGHT_ENABLE
+#    include "backlight.h"
+#endif
```

Comment:
The `QMK_KEYBOARD_H` include already includes this block if backlight is enabled.

```suggestion
```